### PR TITLE
add tabindex to collapsible trigger

### PR DESCRIPTION
**Pull request by jaredlevi**
Added tabindex to the collapsible trigger to make it more friendly for keyboard users.

### CODE REVIEW

File: src/Collapsible.js [hunk 1]
```diff
@@ -265,6 +273,7 @@ Collapsible.defaultProps = {
   onClose: () => {},
   onOpening: () => {},
   onClosing: () => {},
+  tabIndex: 0,
```
Comment (karltaylor) [inline src/Collapsible.js]: I think this should be `null`

File: src/Collapsible.js [hunk 2]
```diff
@@ -175,6 +175,13 @@ class Collapsible extends Component {
           className={triggerClassString.trim()}
           onClick={this.handleTriggerClick}
           style={this.props.triggerStyle && this.props.triggerStyle}
+          onKeyPress={(event) => {
+            const { key } = event;
+              if (key === ' ' || key === 'Enter') {
+                this.handleTriggerClick(event);
+              }
+            }}
+            tabIndex={this.props.tabIndex}
```
Comment (karltaylor) [inline src/Collapsible.js]: and this should be `tabIndex={this.props.tabIndex && this.props.tabIndex}`

Comment (jaredlevi) [reply src/Collapsible.js]: Yup, Updated this. Thanks for the feedback.

**Comment by jaredlevi** [comment]
I updated the README as well.

**Comment by karltaylor** [comment]
@jaredlevi Thanks mate I'll get this merged and released tomorrow 👍 🤓

**Comment by ab1303** [comment]
Hi @jaredlevi , @karltaylor ,
When this piece of code for adding tabIndex was merged... one more piece regarding "onKeyPress" has also been added I presume (inadvertently)...

Now this is causing trouble in my case where I **Inline Edit** trigger

image

**Comment by glennflanagan** [comment]
Hey @ab1303,

I think you may be right in your case, but I’m wondering if you can add an `event.stopPropagation()` on your input field to stop the key press event bubbling. Will that work? 

If not feel free to open a Pull Request.

**Comment by ab1303** [comment]
Thanks @glennflanagan ... I feel that would work ...let me try that... !

**Pull request merged**